### ISSUE TITLE
Accessibility update: Consider adding $ and other info to table cells

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#52594]
## Issue Description
Laura Flannery suggested adding more information to table cells to help screen reader users understand what the cell data means. She suggested:
- Repeating $ in all cells with dollar amounts
- Repeating "Age" or "years" in cells with age ranges

We need to assess the impact of this style update across our tables, decide if other info is useful in cells, and make any needed updates consistently.

---
## Tasks
- [ ] Assess adding info to table cells
- [ ] Document new rules for info to include in cells
- [ ] Make updates to all affected tables
- [ ] Copyedit and publish updates (Megan)

## Acceptance Criteria
- [ ] Tables are updates and new rules are documented

**Comment by sara-amanda** [comment]
@laurwill  is this something @coforma-jamie can assist with?

**Comment by coforma-jamie** [comment]
I can help evaluate this! Could you provide an example of a table with the old pattern?

**Comment by laurwill** [comment]
Hi @coforma-jamie , this [VALife insurance page](https://www.va.gov/life-insurance/options-eligibility/valife/) has the old pattern in the rates tables in the accordions.

**Comment by coforma-jamie** [comment]
Hi @laurwill, reviewed this a bit.

I definitely recommend adding "$" to dollar amounts. In VoiceOver, adding "$" gives the string some context.

Without "$":
> Monthly premium rate for $30,000 coverage (in U.S. dollars), **32 point 70**.  Row X of Y.

With "$":
> Monthly premium rate for $30,000 coverage (in U.S. dollars), **32 dollars and 70 cents**. Row X of Y.

Adding "$" tells VO that we're talking about dollars, and so it reads the number off as dollars/cents and not a standard decimal. That seems very helpful to me! It reads similarly in other screen readers ([see this Deque article](https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/#currency)).

As for "age" or "years," I think that's a bit less clear. Compare these two VoiceOver read-outs:

> Age when you apply, 31. Row X of Y.
> Age when you apply, 31 years. Row X of Y.

I can really see that going either way. It's probably not necessary given [these examples on WebAIM](https://webaim.org/techniques/tables/data). The number "31" doesn't read out any differently because you've added "years" to it, like a dollar amount does.

**So in short:** $ yes, age/years (probably) not.

I hope this is helpful, let me know if you have any questions!

**Comment by laurwill** [comment]
@coforma-jamie that's very helpful! I agree that the benefit of adding $ is clear, so we'll make that our style going forward. But we don't need to worry about ages, etc.

@strelichl could you open a content backlog ticket to add $ to table cells in existing rates pages? This can be a "chipper" priority ticket that content people can pick up as time allows. Steps would be auditing pages to get a list of tables that need this update, updating the tables in drupal, tagging copy editor to review and publish. We can also note this update in all current and future tickets where we're updating rates, so we make this change at the same time.

And @megzehn I'm not sure if this would go in the style guide or in the design system table component page. Maybe add to list of style guide updates for now?

**Comment by sara-amanda** [comment]
Changing priority from `Low` to `Chipper` per @laurwill comments above: 

> This can be a "chipper" priority ticket that content people can pick up as time allows. Steps would be auditing pages to get a list of tables that need this update, updating the tables in drupal, tagging copy editor to review and publish. We can also note this update in all current and future tickets where we're updating rates, so we make this change at the same time.
> 
> And @megzehn I'm not sure if this would go in the style guide or in the design system table component page. Maybe add to list of style guide updates for now?

@coforma-jamie since you did work on this ticket, feel free to self-assign, so that you receive credit for this work. : ) 

@coforma-terry @strelichl

**Comment by megzehn** [comment]
This guidance is now [updated in the style guide](https://design.va.gov/content-style-guide/numbers#signs-and-symbols). 

These are some places on live pages where we need to update the table cells:

First priority (current rates)

- https://www.va.gov/disability/parent-dic-rates/
- https://www.va.gov/disability/survivor-dic-rates/
- https://www.va.gov/pension/veterans-pension-rates/
- https://www.va.gov/pension/survivors-pension-rates/
- https://www.va.gov/pension/protected-pension-rates/

Second priority (past rates for each of these)

I'm going to chat with the content team and see if anyone has bandwidth to pick this up in the upcoming weeks.

**Comment by coforma-jamie** [comment]
The Content team stills need to update some pages with tables in Drupal to follow this decision. There are a LOT of these, so this work got backlogged.

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mistymg** [comment]
Closed per Beth and Randi. Ones that weren't fixed are being moved out of tables.
